base/server/src/main/java/com/netscape/cmscore/ldapconn/PKISocketFactory.java

```diff
@@ -96,6 +98,15 @@ public void init(PKISocketConfig config) {
                     config = engine.getConfig().getSocketConfig();
                 }
                 keepAlive = config.isKeepAlive();
+		try {
+                    mClientCiphers = config.getClientCiphers();
+                } catch (Exception econf) {
+                    // handled below
+                }
+                if ((mClientCiphers != null) && !mClientCiphers.trim().isEmpty())
```

Comment:
Any reason not to just set mClientCiphers to mClientCiphers.trim() ?

Reply:
It seems to be how other code is doing it.  Is there a reason to set mClientCiphers to mClientCiphers.trim() ?

Reply:
I thought it might be more convenient since later you do a check for contents in that variable and do the trim over and over.